config/rbo.yml

```diff
@@ -0,0 +1,27 @@
+# PURL configuration for http://purl.obolibrary.org/obo/rbo
+
+idspace: RBO
+base_url: /obo/rbo
+
+products:
+- rbo.owl: https://raw.githubusercontent.com/DanBerrios/RBO/master/rbo.owl
```

Comment:
Note that these two `products` will not resolve as specified here, because there are no `rbo.owl` or `rbo.obo` files at the top level of your repository.

Reply:
Related to https://github.com/DanBerrios/RBO/issues/64

Reply:
Thanks James!  I had to config my local git ignorecase to be false. I've done this and now I have added rbo.owl the GH repo.  Anything else needed?

Reply:
Thanks! This is fixed, and everything else looks good.